{
  OperationSetBasicTelephony telephony=mainFrame.getTelephony(pps);
  try {
    telephony.answerCallParticipant(callParticipant);
  }
 catch (  OperationFailedException e) {
    logger.error("Could not answer to : " + callParticipant + " caused by the following exception: "+ e);
  }
}
